### PR TITLE
Add a label for leisure-dog_park

### PR DESCRIPTION
**Pull request by jaller94**
Dog parks are currently not translated in the app and show up as `leisure-dog_park`.

I suggest to use the word "dog area" because often those dog parks are part of a bigger park.

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -2635,6 +2635,10 @@ sk:^Vykopávky
 sw:^Machimbo ya kihistoria|kivutio|utalii
 fa:سایت باستان شناسی
 
+leisure-dog_park
```
Comment (mpimenov) [inline data/categories.txt]: Please add other languages too.

File: data/categories.txt [hunk 2]
```diff
@@ -2635,6 +2635,10 @@ sk:^Vykopávky
...
+en:3^dog area
```
Comment (mpimenov) [inline data/categories.txt]: s/dog/Dog/

**Comment by mpimenov** [comment]
Please sign the CLA: https://maps.me/cla/

**Comment by jaller94** [comment]
Is there a translation project? I am not able to verify translations into any other languages.

**Pull request merged**